PINCache/PINApplicationBackgroundTask.h

```diff
@@ -0,0 +1,15 @@
+//
+//  PINApplicationBackgroundTask.h
```

Comment:
Should this be marked as a private header? Background tasks used to be an implementation detail but now we're exposing this class and protocol publicly.

Not sure if this is a big concern, but it does seem like an implementation detail that doesn't have to be public.